app/GraphQL/Ecosystem/Mutations/AccessControlList/UserAccessControlList.php

```diff
@@ -0,0 +1,41 @@
+<?php
+declare(strict_types=1);
+namespace App\GraphQL\Ecosystem\Mutations\AccessControlList;
+
+use App\GraphQL\Ecosystem\Mutations\AccessControlList\Assign;
+use Kanvas\Users\Models\Users;
+
+class UserAccessControlList
+{
+    /**
+     * assignRoleToUser
+     *
+     * @param  mixed $rootValue
+     * @param  array $request
+     * @return void
+     */
+    public function assignRoleToUser($rootValue, array $request): bool
+    {
+        $assign = new Assign(
+            Users::findOrFail($request['user_id']),
```

Comment:
@FredPeal isn't it firstOrFail ? also we need to make sure the user exist within the compay